.github/workflows/docs.yml

```diff
@@ -18,7 +18,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
```

Comment:
I think docs has to be on 20.04 for doxygen

Reply:
Yes, docs needs to be on 20.04 please, we need doxygen >= 1.8.15